src/IntegrationUtils.ts

```diff
@@ -509,4 +550,33 @@ export function killChildProcessForIntegration(integration:string) : Promise<boo
 		}
 		resolve(childProcess ? childProcess.killed : true);
 	});	
-}
\ No newline at end of file
+}
+
+function findChoiceFromStartsWith(inChoice: string | undefined) : string | undefined {
+	if (inChoice) {
+		if (devModeIntegration.startsWith(inChoice)) {
+			return devModeIntegration;
+		}
+		if (basicIntegration.startsWith(inChoice)) {
+			return basicIntegration;
+		}
+	}
+	// other choices not supported at present because they will require additional inputs
+	return undefined;
+ }
+
+ export async function getCurrentFileSelectionPath(): Promise<vscode.Uri> {
```

Comment:
Can you clarify what you're saying here?

Reply:
on master:
- have a not-Camel K artefacts (readme.md for instance) opened in editor
- open command palette
- "Start Apache Camel Integration"
- "Start in Dev mode"
- `Starting new integration via Kamel executable. ` is logged in output channel but nothing is deployed
with this PR:
- have a not-Camel K artefacts (readme.md for instance) opened in editor
- open command palette
- "Start Apache Camel Integration"
- "Start in Dev mode"
- the not-Camel K files is tried to be deployed. it is taking times and resources on the cluster to try to start it.

Reply:
Ok, I think I have this resolved now.

Reply:
Testing via command palette didn't occur to me. I usually just test via the right-click menu and the didact link, which both worked. Now we do a bit more checking to ensure that things occur correctly. 

Note that you will still see "Starting new integration via Kamel executable" because that message occurs earlier than the startIntegration call itself, but the next line will be `Kamel `execution return code: No valid file specified to start integration function call `